test/commands/bot-configs/list.test.ts

```diff
@@ -0,0 +1,168 @@
+      const firstRow = lines[2].split(/\s+/)
+      expect(headers).to.deep.equal([
+        'ConfigId',
+        'ContextTag',
+        'ParentId',
+        'HasInterface',
+        'CreateTime'
+      ])
+      expect(firstRow).to.deep.equal(['456', 'A3_C1', '92', 'true', 'now'])
+    })
+  
+    test
+      .nock(serverURL, (api) =>
+        api
+          .get(endpoint)
+          .query({
+            liveOnly: false,
+            excludeOverrides: false,
+          })
+          .reply(200, botConfigsListResponse)
+      )
+      .stdout()
+      .command(["bot-configs:list", "-B", botId, "--json"])
+      .it("bot-configs:list provides JSON output for given bot", (ctx) => {
+        const result = JSON.parse(ctx.stdout)
+        expect(result).to.deep.equal(botConfigsListResponse)
+      })
+  
+    test
+      .nock(serverURL, (api) =>
+        api
+          .get(endpoint)
+          .query({
+            liveOnly: false,
+            excludeOverrides: false,
+          })
+          .reply(200, botConfigsListResponse)
+      )
+      .stdout()
+      .stderr()
+      .command(["bot-configs:list", "-B", botId, "--csv"])
+      .it("bot-configs:list provides CSV output for given bot", (ctx) => {
+        const [headers, ...results] = ctx.stdout.trim().split('\n')
+        expect(headers.split(',')).to.deep.equal([
+          'ConfigId',
+          'ContextTag',
+          'ParentId',
+          'HasInterface',
+          'CreateTime'
+        ])
+        expect(results[0].split(',')).to.deep.equal(['456', 'A3_C1', '92', 'true', 'now'])
+      })
+
+    test
+      .nock(serverURL, (api) =>
+        api
+          .get(endpoint)
+          .query({
+            liveOnly: false,
+            excludeOverrides: false,
+            appId: "app_123"
+          })
+          .reply(200, botConfigsListResponse)
+      )
+      .stdout()
+      .command(["bot-configs:list", "-B", botId, "--with-runtime-app", "app_123"])
+      .it("bot-configs:list provides human-readable output for given bot and runtime application")
+
+      test
+      .nock(serverURL, (api) =>
+        api
+          .get(endpoint)
+          .query({
+            liveOnly: false,
+            excludeOverrides: false,
+            tag: "A35_C"
+          })
+          .reply(200, botConfigsListResponse)
+      )
+      .stdout()
+      .command(["bot-configs:list", "-B", botId, "--with-tag", "A35_C"])
+      .it("bot-configs:list provides human-readable output for given bot and context tag")
+
+    test
+      .nock(serverURL, (api) =>
+        api
+          .get(endpoint)
+          .query({
+            liveOnly: true,
+            excludeOverrides: false,
+          })
+          .reply(200, botConfigsListResponse)
+      )
+      .stdout()
+      .command(["bot-configs:list", "-B", botId, "--live-only"])
+      .it("bot-configs:list provides human-readable output of bot configurations for given bot that are currently deployed")
```

Comment:
reword to:

'bot-configs:list provides human-readable output of currently deployed bot configurations for given bot'